src/main/java/dev/hephaestus/glowcase/util/NetworkingUtil.java

```diff
@@ -0,0 +1,18 @@
+package dev.hephaestus.glowcase.util;
+
+import dev.hephaestus.glowcase.block.GlowcaseBlock;
+import net.minecraft.server.network.ServerPlayerEntity;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.math.ChunkPos;
+
+public class NetworkingUtil {
+    private NetworkingUtil() {
+    }
+
+    public static boolean cantEditGlowcase(ServerPlayerEntity player, BlockPos pos, GlowcaseBlock glowcase) {
```

Comment:
I know Intellij disagrees with me here, but I think it's more readable to have it be "canEditGlowcase" (even if that means it's always inverted)